### PR TITLE
updated hyperparams by increasing trial, fix batch size

### PR DESCRIPTION
**Pull request by wiinnie9**

### CODE REVIEW

File: train/uft/tuning.py [hunk 1]
```diff
@@ -50,7 +50,6 @@ def get_params(trial):
     discriminator_adam_lr = trial.suggest_float('discriminator_adam_lr', 1e-6, 1e-2, log = True)
     denoiser_loss_b1 = trial.suggest_float('denoiser_loss_b1', 0.1, 0.9)
     denoiser_loss_b2 = 1 - denoiser_loss_b1
-    print_loss_interval = trial.suggest_int('print_loss_interval', 50, 500)
     denoiser_adversarial_loss_clip_max = trial.suggest_float('denoiser_adversarial_loss_clip_max', 0.0, 1.0)
     denoiser_adversarial_loss_clip_min = trial.suggest_float('denoiser_adversarial_loss_clip_min', 0.0, 1.0)
     calc_eval_loss_interval = trial.suggest_int("calc_eval_loss_interval", 50, 500)
```
Comment (superleesa) [inline train/uft/tuning.py]: this doesn't need to be a parameter either, as it's simply used for recording purpose (does not affect the training result)

File: train/uft/tuning.py [hunk 2]
```diff
@@ -50,7 +50,6 @@ def get_params(trial):
     discriminator_adam_lr = trial.suggest_float('discriminator_adam_lr', 1e-6, 1e-2, log = True)
     denoiser_loss_b1 = trial.suggest_float('denoiser_loss_b1', 0.1, 0.9)
     denoiser_loss_b2 = 1 - denoiser_loss_b1
-    print_loss_interval = trial.suggest_int('print_loss_interval', 50, 500)
     denoiser_adversarial_loss_clip_max = trial.suggest_float('denoiser_adversarial_loss_clip_max', 0.0, 1.0)
     denoiser_adversarial_loss_clip_min = trial.suggest_float('denoiser_adversarial_loss_clip_min', 0.0, 1.0)
```
Comment (superleesa) [inline train/uft/tuning.py]: - i think clip_min can be negative.
- we need to ensure that clip_min < clip_max always but with this setting, this is not guarantee always and might confuse the model.

i think for now, we can set these parameters to None (we don't clip adversarial loss) because i'm not yet sure how effective this clipping will be. later on, we can add this.

File: train/uft/tuning.py [hunk 3]
```diff
@@ -80,7 +79,7 @@ def objective (trial):
     eval_dir = config_dir["eval_dir"]
 
     num_epochs = trial.suggest_int('num_epochs', 1, 15)
```
Comment (superleesa) [inline train/uft/tuning.py]: i think this can be fixed to 17. (because sho did the early stopping test separately and after around epoch 20, there was no improvement)

File: train/uft/tuning.py [hunk 4]
```diff
@@ -123,7 +122,7 @@ def objective (trial):
 
 def run_tuning():
     study = optuna.create_study(direction="maximize", study_name="uft-tuning")
-    study.optimize(objective, n_trials=5)
+    study.optimize(objective, n_trials=15)
```
Comment (superleesa) [inline train/uft/tuning.py]: i think n_trials is too small? how much time did this take? it depends on the time but i think we want at least 20 trials.

**Review by superleesa — CHANGES_REQUESTED** [review]
i'm sorry for late review... i think we still need to change some hyper-paramter settings

**Comment by superleesa** [comment]
@weeenieee

File: train/uft/tuning.py (hunk 1 above)
Comment (wiinnie9) [reply train/uft/tuning.py]: ok

File: train/uft/tuning.py (hunk 2 above)
Comment (wiinnie9) [reply train/uft/tuning.py]: ok

File: train/uft/tuning.py (hunk 3 above)
Comment (wiinnie9) [reply train/uft/tuning.py]: ok

File: train/uft/tuning.py (hunk 4 above)
Comment (wiinnie9) [reply train/uft/tuning.py]: i can try fixing epochs to 17 and then adding trials to 25.

Comment (superleesa) [reply train/uft/tuning.py]: thanks!

File: train/uft/best_params.yml [hunk 5]
```diff
@@ -1,7 +1,4 @@
-calc_eval_loss_interval: 200
-denoiser_adam_lr: 0.00012090676833404486
-denoiser_adversarial_loss_clip_max: 0.7253553053728639
-denoiser_adversarial_loss_clip_min: 0.42959290591546195
-denoiser_loss_b1: 0.654205478362656
-discriminator_adam_lr: 0.009627422411867004
-num_epochs: 11
+calc_eval_loss_interval: 381
```
Comment (superleesa) [inline train/uft/best_params.yml:1]: calc_eval_loss_interval isn't a tuneable parameter so can remove. but we don't need to run the hyper-parameter optimization again as it's not likely to change the result.

File: train/uft/best_params.yml [hunk 6]
```diff
@@ -1,7 +1,4 @@
...
+denoiser_adam_lr: 1.3745488663779329e-05
+denoiser_loss_b1: 0.48803387886673455
+discriminator_adam_lr: 0.0003565004908213629
```
Comment (superleesa) [inline train/uft/best_params.yml:4]: i think overall the result makes sense

- denoiser_adam_lr < discriminator_adam_lr:  probably because more parameters to tune and sensitive to even a small change. 10^-5 makes sense as well.
- denoiser_loss_b1: about half, which is fair (but it seems like slightly less important than b2, which is interesting)

**Review by superleesa — APPROVED** [review]
calc_eval_loss_interval isn't a tuneable parameter so can remove. but we don't need to run the hyper-parameter optimization again as it's not likely to change the result. i think we can merge after this change!

File: train/uft/best_params.yml (hunk 5 above)
Comment (wiinnie9) [reply train/uft/best_params.yml:1]: ok updated, last time did not work but now can already
